src/main/proto/com/craxiom/messaging/cdma_record.proto

```diff
@@ -31,6 +31,15 @@ message CdmaRecordData {
     int32 groupNumber = 9;
     int32 accuracy = 10;
 
+    // Proposed new common fields to support orientation of sensor and
+    // sensor parameters.  All fields are optional.
+    float rotationX = 50;
+    float rotationY = 51;
+    float rotationZ = 52;
+    bool directionalSensor = 53;
```

Comment:
I don't see the `directionalSensor` field in the AsyncAPI yaml spec. Is this needed since we have the field of view?

Reply:
directionalSensor was a vestige of an older iteration.  I have eliminated all references to it.